spec/controllers/slides_controller_spec.rb

```diff
@@ -98,7 +98,7 @@
       login_by_user default_user
       post :create, params: { slide: data.attributes }
       expect(response.status).to eq(200)
-      expect(response).to render_template 'slides/aws/new'
+      expect(response).to render_template "slides/#{CloudConfig::service_name}/new"
```

Comment:
__[RuboCop]__

Style/ColonMethodCall: Do not use `::` for method calls.


<!-- Comment by [SideCI](https://sideci.com) -->